README.md

```diff
@@ -2,7 +2,7 @@ Libero Article Store
 ====================
 
 [![Build Status](https://travis-ci.com/libero/article-store.svg?branch=master)](https://travis-ci.com/libero/article-store)
-[![Build Status](https://github.com/libero/article-store/workflows/CI/badge.svg?branch=master)](https://github.com/libero/article-store/actions?query=branch%3Amaster)
+[![Build Status](https://github.com/libero/article-store/actions?query=branch%3Amaster+workflow%3ACI)](https://github.com/libero/article-store/actions?query=branch%3Amaster)
```

Comment:
I think the link to the image is broken:
[![Build Status](https://github.com/libero/article-store/actions?query=branch%3Amaster+workflow%3ACI)](https://github.com/libero/article-store/actions?query=branch%3Amaster)


```suggestion
[![Build Status](https://github.com/libero/article-store/workflows/CI/badge.svg?branch=master)](https://github.com/libero/article-store/actions?query=branch%3Amaster+workflow%3ACI)
```

[![Build Status](https://github.com/libero/article-store/workflows/CI/badge.svg?branch=master)](https://github.com/libero/article-store/actions?query=branch%3Amaster+workflow%3ACI)

Reply:
I changed the wrong one!